public/admin/js/AngularJS/main.js

```diff
@@ -86,6 +86,7 @@ MyApp.controller('MyController', ['$scope', '$http', '$log', 'Upload', '$timeout
                 age: $scope.member.age,
                 avatar: $scope.file,
             };
+            // console.log($scope.file);
```

Comment:
Please remove console.log(), dd() and var_dump() lines. Debugging lines should not be committed.